.github/workflows/tox.yaml

```diff
@@ -0,0 +1,17 @@
+name: tox
+on:
+  pull_request:
+jobs:
+  venv:
+    runs-on: ubuntu-latest
+    permissions: {}
+    container:
+      image: quay.io/cockpit/tasks:latest
```

Comment:
Could we rather use the unittests container? I'm a bit hesitant to pull `tox` into the tasks container, as it's so annoyingly big:

```
total download size: 147 M
Installed size: 721 M
```
This also [caused the failure](https://github.com/cockpit-project/cockpit/actions/runs/5447338488/jobs/9909210340?pr=19054).

*if* you move it into the unit-tests container, then the whole workflow should move to unit-tests.yaml as a new job, and also into unit-tests-refresh.yaml. FTR, I'm also fine with your previous approach of just `apt install`ing it.

Reply:
I've created https://github.com/allisonkarlitskaya/toxbox